ocs_ci/deployment/ibm.py

```diff
@@ -26,206 +30,174 @@ def destroy_ocs(self):
-            for sc in sclist:
-                if sc in scnoprovisioner:
-                    if sc not in tmplist:
-                        tmplist.append(sc)
-
-        scandlvlist = []
-        for sc in tmplist:
-            storage_class = ocp.OCP(kind="StorageClass", resource_name=sc)
-            localvolumemeta = storage_class.get().get("metadata")
-            localvolume = localvolumemeta["labels"][
-                "local.storage.openshift.io/owner-name"
-            ]
-            scandlvlist.append((sc[0], localvolume))
-
-        alldevicepaths = []
-        for lv in scandlvlist:
-            # Get localvolume and corresponding devices
-            localvolume = (
-                ocp.OCP(
-                    kind="LocalVolume",
-                    namespace=local_storage_namespace,
-                    resource_name=lv[1],
-                )
-                .get()
-                .get("spec")
+        # https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.5/html/deploying_openshift_container_storage_using_bare_metal_infrastructure/assembly_uninstalling-openshift-container-storage_rhocs
+
+        # Section 3.1 Step 1
+        # Deleting PVCs
+        rbd_pvcs = [
+            p
+            for p in pvc.get_all_pvcs_in_storageclass("ocs-storagecluster-ceph-rbd")
+            if not (
+                p.data["metadata"]["namespace"] == cluster_namespace
+                and p.data["metadata"]["labels"]["app"] == "noobaa"
             )
-            scdevices = localvolume["storageClassDevices"]
-            devicepaths = [sub["devicePaths"] for sub in scdevices]
-            alldevicepaths.append(devicepaths[0][0])
-
-        # STEP 5
-        # delete storage cluster - all of them.
-        scdelete = f"oc delete -n {cluster_namespace} storagecluster --all --wait=true"
-        logger.info(scdelete)
-        out = run_cmd(scdelete)
-        logger.info(out)
-
-        # STEP 6
-        spdelete = f"oc delete project {cluster_namespace} --wait=true --timeout=5m"
-        logger.info(spdelete)
-        out = run_cmd(spdelete)
-        logger.info(out)
-
-        default = "oc project default"
-        logger.info(default)
-        out = run_cmd(default)
-        logger.info(out)
-
-        # STEP 7 & 9
-        nodes = ocp.OCP(kind="node").get().get("items", [])
-        worker_nodes = [
-            node
-            for node in nodes
-            if "cluster.ocs.openshift.io/openshift-storage"
-            in node["metadata"]["labels"]
         ]
-        worker_node_names = [node["metadata"]["name"] for node in worker_nodes]
-        logger.warning(worker_node_names)
-
-        for worker_node in worker_node_names:
-            rookstring = (
-                "oc debug node/"
-                + worker_node
-                + " -- chroot /host rm -rfv /var/lib/rook"
+        pvc.delete_pvcs(rbd_pvcs)
+        cephfs_pvcs = pvc.get_all_pvcs_in_storageclass("ocs-storagecluster-cephfs")
```

Comment:
can you use from constants here? https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/constants.py#L96